lib/3scale/api/client.rb

```diff
@@ -623,6 +623,107 @@ def delete_service_feature(service_id, id)
         true
       end
 
+      # @api public
+      # @param [Fixnum] account_id Account ID
+      # @param [String] state State
+      # @param [String] role Role
+      # @return [Array<Hash>]
+      def list_users(account_id, state: nil, role: nil)
+        params = { state: state, role: role }.reject { |_, value| value.nil? }
+        response = http_client.get("/admin/api/accounts/#{account_id}/users", params: params)
+        extract(collection: 'users', entity: 'user', from: response)
+      end
+
+      # @api public
+      # @param [String] email User email
+      # @param [String] username User Username
+      # @param [String] password User password
+      # @param [Hash] attributes User Attributes
+      # @return [Hash]
+      def create_user(attributes = {}, account_id:, email:, username:, password:, **rest)
```

Comment:
Why not simpler signature?

```
def create_user(account_id:, email:, username:, password:, **rest)
        body = { email: email, username: username, password: password }.merge(rest)
        response = http_client.post("/admin/api/accounts/#{account_id}/users", body: body)
        extract(entity: 'user', from: response)
end
```

Reply:
Thanks for the feedback. For the create_user method I took a similar signature as the signup method:
def signup(attributes = {}, name:, username:, **rest)

Reply:
If you see any advantage doing so, please explain to me. Happy to learn.

Otherwise, let's keep it simple.

Reply:
I simplified the signature

Reply:
```

Reply:
done, thanks for the review